src/elements/content-picker/__tests__/Footer.test.js

```diff
@@ -25,6 +25,15 @@ describe('elements/content-picker/Footer', () => {
             expect(wrapper.find('.footer-child').length).toBe(1);
         });
 
+        test('should render Footer tooltips with aria-label', () => {
+            const tooltipButtonWrapper = getWrapper()
+                .find('Tooltip')
+                .find('Button');
```

Comment:
you could probably call this `buttons = getWrapper()` since tooltipButton might imply a button inside of the tooltip itself

I think you could also get away with removing `.find('Tooltip')`